src/Scientist/Internals/Experiment.cs

```diff
@@ -10,7 +10,7 @@ internal class Experiment<T, TClean> : IExperiment<T, TClean>, IExperimentAsync<
 
         private static readonly Func<Task<bool>> _alwaysRun = () => Task.FromResult(true);
         private static readonly Action<Operation, Exception> _alwaysThrow
-            = (operation, exception) => { throw exception; };
+            = (operation, exception) => throw new OperationException(operation, exception);
```

Comment:
It looks like AppVeyor is barfing here.  I think throw expressions were added in C# 7.0, which isn't yet available in this project.  Adding the brackets back here should resolve that issue.